metadata/src/main/java/org/apache/kafka/controller/ConfigurationControlManager.java

```diff
@@ -303,6 +339,14 @@ private ApiError validateAlterConfig(ConfigResource configResource,
             if (alterConfigPolicy.isPresent()) {
                 alterConfigPolicy.get().validate(new RequestMetadata(configResource, alteredConfigsForAlterConfigPolicyCheck));
             }
+            if (featureControl.isElrFeatureEnabled()) {
+                if (configResource.type() == Type.BROKER && !configResource.name().isEmpty()) {
+                    for (Entry record : allConfigs.entrySet()) {
```

Comment:
You should specify the key and value type for Entry. It is a generic class.

Reply:
Specified.